### PR TITLE
fix(dependencies): Bump dependencies

### PR DESCRIPTION
**Pull request by wizzy25**
Bump dependencies whose older versions contain vulnerabilities

### CODE REVIEW

File: src/spec/integration/discounts.spec.coffee [hunk 1]
```diff
@@ -24,6 +24,7 @@ newCartDiscount = ->
     predicate: 'variant.id = 1'
   sortOrder: "0.#{_.reduce [1..orderCount], ((m, i) -> '' + m + i), '2'}"
   isActive: false
+  requiresDiscountCode: true
```
Comment (PhilippSpo) [inline src/spec/integration/discounts.spec.coffee]: Doesn't look like a dependency bump to me 🔍😜 Can you clarify what this does/why it is needed quickly?

Comment (wizzy25) [reply src/spec/integration/discounts.spec.coffee]: I also realised builds were failing on Master with this error: 
It seems to be a change on the API, that if you don't specify `requiresDiscountCode` to `true` in a cart discount, then you cannot assign discount codes to it
```
Failures:
  1) Integration Cart Discounts should create a discount code and apply it to a cart
   Message:
     BadRequest: Cart Discount '78836da8-622e-4215-b941-5e5237b0da39' doesn't require a discount code.
   Stacktrace:
     BadRequest: Cart Discount '78836da8-622e-4215-b941-5e5237b0da39' doesn't require a discount code.
    at /home/travis/build/sphereio/[secure]/lib/services/base.js:9:33989
    at DiscountCodeService.BaseService._wrapResponse (/home/travis/build/sphereio/[secure]/lib/services/base.js:9:34939)
    at /home/travis/build/sphereio/[secure]/lib/services/base.js:9:26503
    at Request._callback (/home/travis/build/sphereio/[secure]/lib/connect/rest.js:9:10046)
    at Request.self.callback (/home/travis/build/sphereio/[secure]/node_modules/request/request.js:360:22)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)
    at Request.<anonymous> (/home/travis/build/sphereio/[secure]/node_modules/request/request.js:1202:14)
    at emitOne (events.js:82:20)
    at Request.emit (events.js:169:7)
    at IncomingMessage.<anonymous> (/home/travis/build/sphereio/[secure]/node_modules/request/request.js:1150:12)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:923:12)
    at nextTickCallbackWith2Args (node.js:511:9)
    at process._tickCallback (node.js:425:17)
Finished in 61.168 seconds
2274 tests, 4709 assertions, 1 failure, 0 skipped
```

Comment (PhilippSpo) [reply src/spec/integration/discounts.spec.coffee]: Thanks for clarifying. Is that a change that should be mentioned in the release notes—as a commit message I guess?

Comment (wizzy25) [reply src/spec/integration/discounts.spec.coffee]: I'll add it to the merge commit message. As it was a test, I didn't initially mention it.

I don't know if I should also mention it in the release notes

**Pull request merged**